basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -1269,6 +1269,14 @@ namespace basic_autonomy
                 autoware_point.x = trajectory_points[i].x;
                 autoware_point.y = trajectory_points[i].y;
                 autoware_point.longitudinal_velocity_mps = lag_speeds[i];
+                double yaw = 0.0;
+                if (i<= max_size-1)
+                {
+                    double yaw = std::atan2(trajectory_points[i+1].y - trajectory_points[i].y, trajectory_points[i+1].x - trajectory_points[i].x);
+                    // the last trajectory point does not have a next point and its yaw cannot be calculated. But since the new trajectories are being generated frequently thats not an issue.
```

Comment:
Honestly I dont think that will happen, since when the car starts moving after stopping it must have received a new trajectory.
but still implemented your suggestion and added a comment